assets/js/page.js

```diff
@@ -0,0 +1,12 @@
+function setPermalinks () {
+  var headers = document.querySelectorAll('.tm-container h4')
+
+  // Apply permalinks to headers
+  if (headers.length > 0) {
+    Array.prototype.forEach.call(headers, function (el, i) {
```

Comment:
@ahmadnassri I cannot use ES6 because we don't run Babel on this project. Also, is a 5kb dependency really needed for something I was able to do in a few lines? Right now I only targeted `h4`s, but it's trivial to add more selectors if we need them later.